pvnet_app/forecast_compiler.py

```diff
@@ -169,6 +170,44 @@ def predict_batch(self, batch: NumpyBatch) -> None:
         # Log max prediction
         self.log_info(f"GSP IDs: {these_gsp_ids}")
         self.log_info(f"Max prediction: {np.max(preds, axis=1)}")
+    
+    def validate_forecast(
+        max_forecast_mw: float,
```

Comment:
I wonder if you could take in all forecast values, and then take in the max in this function. This is becasue I can imagine other validations that use all the valuations, not just the maximum